src/Raven.Server/Properties/Settings/settings.posix.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "ServerUrl": "http://localhost:8080",
-  "RunInMemory": false
+    "ServerUrl": "http://127.0.0.1:0",
```

Comment:
Shouldn't we also specify a well known path here?

Reply:
We don't know a good path with proper permissions. Windows' `/raven` won't do here.

Reply:
`$home/raven-dbs/`?

Reply:
hmm most apps on linux just use `~/.appname` (~ would probably need to be expanded by us) when it comes to saving local app data, but then it becomes hidden in unix fs sense. 

Once it's set up system-wide however most appropriate dir would be e.g. `/var/lib/raven`. See https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s08.html and https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html .

So answer is it depends. The best default to which we have access to is stuff in home dir as you pointed out. Since we have `C:\Raven` on Windows, we could go with `~/raven` for Linux.

Reply:
Note that `C:\Raven` also requires admin privileges on most sytesm